gcpvault.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/cenkalti/backoff"
```

Comment:
If the goal is to use v2, this should be `"github.com/cenkalti/backoff/v2"`

Reply:
the package `github.com/cenkalti/backoff/v2` doesn't exist.
```
$ go build
go: finding module for package github.com/cenkalti/backoff/v2
gcpvault.go:7:2: module github.com/cenkalti/backoff@latest found (v2.2.1+incompatible), but does not contain package github.com/cenkalti/backoff/v2
```

Reply:
ah. Looks like the author of the library is using the branch strategy for mod versions. I'm sorry I haven't worked with that as much because it can break non-module backwards compatibility, the go get command is `go get github.com/cenkalti/backoff@v2`, but you are totally right about the import as it is. 

I changed it to `import "github.com/cenkalti/backoff/v2.2.1"` and `gofmt` autocorrected it to `import "github.com/cenkalti/backoff"`

It looks like I was confused about the import. Sorry about that.

Reply:
This import should go into non-stdlib imports block down starting line 12.